plugins/linode/personal_access_token.go

```diff
@@ -31,9 +32,32 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
 		Importer: importer.TryAll(
 			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryConfigFile(),
 		)}
 }
 
 var defaultEnvVarMapping = map[string]sdk.FieldName{
 	"LINODE_CLI_TOKEN": fieldname.Token,
 }
+
+// TryConfigFile looks for the token in the ~/.config/linode-cli file.
+func TryConfigFile() sdk.Importer {
+	return importer.TryFile("~/.config/linode-cli", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		configFile, err := contents.ToINI()
+		if err != nil {
+			out.AddError(err)
+			return
+		}
+
+		fields := make(map[sdk.FieldName]string)
+		for _, section := range configFile.Sections() {
+			if section.HasKey("token") && section.Key("token").Value() != "" {
+				fields[fieldname.Token] = section.Key("token").Value()
+			}
+		}
+
+		out.AddCandidate(sdk.ImportCandidate{
```

Comment:
If I understand the format of the configuration file correctly, it can contain multiple credentials. For each credential, we'll want to create a separate 1Password item. Therefore, we should move `out.AddCandidate` inside the loop through the sections, so that we're adding a new entry for each section that contains a token.

Reply:
Consider adding another test fixture for an example with multiple sections (users) to ensure that works as expected.

Reply:
If you help me see where to add in support for name hints and multiple credentials I'll also add a test fixture to test that.

Reply:
The loop will look something like that
```
for _, section := range configFile.Sections() {
	if section.HasKey("token") && section.Key("token").Value() != "" {
		fields[fieldname.Token] = section.Key("token").Value()
                 out.AddCandidate(sdk.ImportCandidate{
		        NameHint: importer.SanitizeNameHint(section.Name()),
		        Fields: map[sdk.FieldName]string{
			        fieldname.Token: section.Key("token").Value(),
		        },
	        })
	}
}
```
Note: you don't need to add candidates outside of the loop anymore.

Each section is considered a separate profile. If it has a `token` property we would create a candidate with the name the same as a section name.

Reply:
I added more comments that might help you with the unit tests.
Please let me know if you need help.

Reply:
That new version of the loop makes sense, I've now added it.